test/performance/broker-rmq/README.md

```diff
@@ -0,0 +1,78 @@
+  --type merge \
+  --patch '{"data":{"ingress.class":"kourier.ingress.networking.knative.dev"}}'
+```
+> Aside: does this look right to you @benmoss ? https://github.com/knative/serving/releases/download/v0.26.1/serving-crds.yaml . I didn't install v0.26.1 - latest release - because of it.
+## 3/5. Install Knative Eventing RabbitMQ
+```
+# Installing RabbitMQ Operator ...
+kubectl apply --filename https://github.com/rabbitmq/cluster-operator/releases/download/v1.9.0/cluster-operator.yml
+# Installing cert-manager ...
+curl -sL https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml \
+| sed 's/kube-system/cert-manager/' \
+| kubectl apply --namespace cert-manager --filename -
+# Installing RabbitMQ Messaging Topology Operator with cert-manager integration ...
+kubectl apply --filename https://github.com/rabbitmq/messaging-topology-operator/releases/download/v1.2.1/messaging-topology-operator-with-certmanager.yaml
+# Installing Knative Eventing RabbitMQ Broker ...
+kubectl apply --filename https://github.com/knative-sandbox/eventing-rabbitmq/releases/download/v0.26.0/rabbitmq-broker.yaml
+```
+In case you want to test the latest dev version of eventing-rabbitmq, follow these steps:
+```
+# ⚠️ TODO after https://github.com/knative-sandbox/eventing-rabbitmq/pull/445 is merged:
+# git clone https://github.com/knative-sandbox/eventing-rabbitmq $GOPATH/src/knative.dev/eventing-rabbitmq
+git clone https://github.com/ikvmw/eventing-rabbitmq $GOPATH/src/knative.dev/eventing-rabbitmq
+cd $GOPATH/src/knative.dev/eventing-rabbitmq
+git checkout -b issue-239 origin/issue-239
+ko apply --filename config/broker
+```
+## 4/5. Run Knative Eventing RabbitMQ benchmarks
+```
+cd $GOPATH/src/knative.dev/eventing-rabbitmq
+ko apply --filename test/performance/broker-rmq/100-broker-perf-setup.yaml
+kubectl wait --for=condition=AllReplicasReady=true rmq/rabbitmq-test-cluster --timeout=10m --namespace perf-eventing
+kubectl wait --for=condition=IngressReady=true brokers/rabbitmq-test-broker --timeout=10m --namespace perf-eventing
+kubectl wait --for=condition=SubscriberResolved=true triggers/rabbitmq-broker-perf --timeout=10m --namespace perf-eventing
+ko apply --filename test/performance/broker-rmq/200-broker-perf.yaml
+```
+## 5/5. Download & visualise Knative Eventing RabbitMQ benchmark results
+Pre-requisite: [gnuplot](http://www.gnuplot.info/) (on macOS it's `brew install gnuplot`)
+```
+kubectl wait --for=condition=complete job/rabbitmq-broker-perf-send-receive --timeout=10m --namespace perf-eventing
+kubectl port-forward rabbitmq-broker-perf-aggregator 10001:10001 --namespace perf-eventing &
+PORT_FORWARD_PID=$!
+cd $GOPATH/src/knative.dev/eventing-rabbitmq/test/performance
+curl http://localhost:10001/results > eventing-rabbitmq-broker-perf-results.csv
+kill $PORT_FORWARD_PID
+wait $PORT_FORWARD_PID 2>/dev/null
+gnuplot -c latency-and-throughput.plg eventing-rabbitmq-broker-perf-results.csv 0.5 0 1100
+```
```

Comment:
Would be useful to have the legend show units as well.
I made these assumptions but it would be better to have it explicitly listed in the README
Y axis
0-1200 (# of messages)
Latency (s)

X axis
Time(time in epoch)

Reply:
There is a lot to be desired with gnuplot. My recommendation would be to look for more cloud-native friendly alternative to gnuplot.